policy-test/src/grpc.rs

```diff
@@ -81,6 +101,22 @@ pub struct OutboundPolicyClient {
     client: OutboundPoliciesClient<GrpcHttp>,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub enum Kind {
```

Comment:
is this a duplicate of the enum in https://github.com/linkerd/linkerd2/blame/main/policy-controller/grpc/src/workload.rs ? can we use that here instead of redefining it?

Reply:
yup! good spot. I originally intended to do it but we needed to import the grpc crate. I went ahead with it 👍🏻